packages/core/src/chains/sol/sdkSol/OffchainMessage.ts

```
@@ -130,6 +138,308 @@
+      result.set(version, offset);
+      offset += version.length;
+      result.set(formatBytes, offset);
+      offset += formatBytes.length;
+      result.set(lengthBytes, offset);
+      offset += lengthBytes.length;
+      result.set(messageBytes, offset);
+
+      return Buffer.from(result).toString('hex');
+    }
+    // Standard format
+    this.validateSignerPublicKeys(signerPublicKeys);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)